playbooks/ansible/roles/sit.cephfs/templates/smb_share.conf.j2

```diff
@@ -1,6 +1,6 @@
 [{{ name }}-{{ config.be.name }}-{{ config.be.variant }}]
 comment = Volume '{{ name }}' from {{ config.be.name }}({{ config.be.variant }})
-vfs objects = acl_xattr
+vfs objects = acl_xattr ceph_snapshots
```

Comment:
I haven't looked into the internals but can there be a requirement for _ceph_snapshots_ to be placed first in the `vfs objects` list when other modules like _acl_xattr_ etc.(other than _ceph_) are present in the stack?

Reply:
I haven't encountered such situation yet. The results with changed order is been same both on the cephfs and cephfs.vfs